demo/auth/authService.js

```diff
@@ -0,0 +1,39 @@
+var fh = require('fh-js-sdk');
```

Comment:
We may move this and wfm module to `components` folder. It feel strange that we have it here and I missed that when reviewing